### PR TITLE
Docs-update

### PR DESCRIPTION
**Pull request by tac0turtle**
creating a draft PR for this work

- [x] add ci deployment
- [x] see if any docs need updatings

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
```
Comment (fedekunze) [inline docs/README.md]:
```suggestion
<!--
```

File: docs/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+parent:
+  order: false
+layout: home
+---
```
Comment (fedekunze) [inline docs/README.md]:
```suggestion
-->
```

File: docs/images/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+parent:
+  order: false
+---
```
Comment (fedekunze) [inline docs/images/README.md]:
```suggestion
-->
```

File: docs/images/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (fedekunze) [inline docs/images/README.md]:
```suggestion
<!--
```

File: docs/post.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```
Comment (fedekunze) [inline docs/post.sh]:
```suggestion
#!/usr/bin/env bash

```

File: docs/versions [hunk 6]
```diff
@@ -0,0 +1 @@
+gamarin/docs-update
```
Comment (fedekunze) [inline docs/versions]: does this need to change later to `master` or a release tag?

File: docs/translations/readme.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (fedekunze) [inline docs/translations/readme.md]:
```suggestion
<!--
```

File: docs/translations/README.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (fedekunze) [inline docs/translations/README.md]:
```suggestion
<!--
```

File: docs/translations/README.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
...
+parent:
+  order: false
+---
```
Comment (fedekunze) [inline docs/translations/README.md]:
```suggestion
-->
```

File: docs/translations/readme.md [hunk 10]
```diff
@@ -0,0 +1,4 @@
...
+parent:
+  order: false
+---
```
Comment (fedekunze) [inline docs/translations/readme.md]:
```suggestion
-->
```

**Review by fedekunze — CHANGES_REQUESTED** [review]

File: docs/versions (hunk 6 above)
Comment (tac0turtle) [reply docs/versions]: mircea still needs to push a final change

File: .github/workflows/linkchecker.yml [hunk 11]
```diff
@@ -0,0 +1,11 @@
+name: Check links
```
Comment (alexanderbez) [inline .github/workflows/linkchecker.yml:1]: Don't we already have a link-checker?

File: docs/images/README.md [hunk 12]
```diff
@@ -0,0 +1,4 @@
+<!--
+parent:
+  order: false
+---
```
Comment (fedekunze) [inline docs/images/README.md]:
```suggestion
-->
```

**Review by fedekunze — APPROVED** [review]

File: Makefile [hunk 13]
```diff
@@ -90,6 +90,27 @@ install: go.sum
...
+	@cd docs && \
+	while read p; do \
+		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
+		mkdir -p ~/output/$${p} ; \
+		cp -r .vuepress/dist/* ~/output/$${p}/ ; \
+		cp ~/output/$${p}/index.html ~/output ; \
+	done < versions ;
+
+sync-docs:
+	cd ~/output && \
```
Comment (alexanderbez) [inline Makefile:107]: This is really only run in the CI process. Does it need to be in the makefile? Most devs won't have or even use `aws`.

File: docs/delegators/delegator-faq.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
----
```
Comment (alexanderbez) [inline docs/delegators/delegator-faq.md:1]: Any updates on having this not rendered in markdown?

File: docs/post.sh (hunk 5 above)
Comment (alexanderbez) [inline docs/post.sh]: Why is this needed?

File: docs/pre.sh [hunk 15]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```
Comment (alexanderbez) [inline docs/pre.sh]: ditto?

File: docs/delegators/delegator-faq.md (hunk 14 above)
Comment (tac0turtle) [reply docs/delegators/delegator-faq.md:1]: `<!---` i believe solves this we are doing it in most places, if we aren't then will change it

File: .github/workflows/linkchecker.yml (hunk 11 above)
Comment (alexanderbez) [reply .github/workflows/linkchecker.yml:1]: ohhh that's the SDK right, if so, ignore.

Comment (tac0turtle) [reply .github/workflows/linkchecker.yml:1]: not in the ci, this is the same github action that is in the cosmos-sdk

File: docs/post.sh (hunk 5 above)
Comment (tac0turtle) [reply docs/post.sh]: I brought this in from the sdk. @fadeev is it required?

Comment (alexanderbez) [reply docs/post.sh]: its an empty script file.

File: Makefile (hunk 13 above)
Comment (mircea-c) [reply Makefile:107]: I don't want to make a separate makefile just to deploy the docs. Nor do I want to add it to the Circle config which is huge as is and could use more trimming.

File: docs/post.sh (hunk 5 above)
Comment (fadeev) [reply docs/post.sh]: Nope, not required.

File: docs/pre.sh (hunk 15 above)
Comment (fadeev) [reply docs/pre.sh]: Not required as well.

**Review by alexanderbez — APPROVED** [review]
ACK

File: docs/translations/readme.md [hunk 16]
```diff
@@ -0,0 +1,4 @@
+<!--
```
Comment (erikgrinaker) [inline docs/translations/readme.md:1]: @marbar3778 Is there any reason for having `readme.md` in addition to `README.md`, or just an oversight? This throws Git warnings on macOS since its filesystem is case-insensitive, and these two filenames therefore map to the same file.

Comment (tac0turtle) [reply docs/translations/readme.md:1]: I think someone pushed this on accident. I am opposed to having empty readme's but i believe they are needed for the vuepress config.
